models/Document/Editable/Date.php

```diff
@@ -74,19 +74,21 @@ public function getDataEditmode()
      */
     public function frontend()
     {
+        if (!$this->date instanceof \DateTimeInterface) {
```

Comment:
Should we check for `Carbon` here? Both, `toIso8601String()` and `formatLocalized()` don't exist in the `DateTimeInterface` but only in `Carbon`.